src/main/java/ch/uzh/ifi/hase/soprafs22/entity/User.java

```diff
@@ -60,6 +60,9 @@ public class User implements Serializable {
     @Column
     private Gender gender;
 
+    @Column
+    private int age;
+
```

Comment:
yes, true, need to change this, but i think it makes sense to somehow keep the age in the user, because passing a parameter list to the query gets tedious, especially when modifiying the query